featuretools/entityset/serialize.py

```diff
@@ -13,10 +13,10 @@
 ps = import_or_none("pyspark.pandas")
 
 FORMATS = ["csv", "pickle", "parquet"]
-SCHEMA_VERSION = "7.0.0"
+SCHEMA_VERSION = "7.1.0"
```

Comment:
i think if we're unable to deserialize an entityset parquet serialized with schema version 7.0.0 we should consider this a breaking change and bump schema version to 8.0.0